### PR TITLE
Add a command-line version of Package.verify

### PR DESCRIPTION
**Pull request by dimaryaz**

### CODE REVIEW

**Comment by dimaryaz** [comment]
I couldn't decide which options should be passed as `--name`, and which should be positional args.

File: api/python/quilt3/main.py [hunk 1]
```diff
@@ -20,6 +20,16 @@ def cmd_config(catalog_url):
         api.config(catalog_url)
 
 
+def cmd_verify(name, registry, top_hash, src, extra_files_ok):
+    pkg = api.Package.browse(name, registry, top_hash)
+    if pkg.verify(src, extra_files_ok):
+        print("Success")
+        return 0
+    else:
+        print("Failed!")
```
Comment (akarve) [inline api/python/quilt3/main.py]:
```suggestion
        print("Verification failed")
```

File: api/python/quilt3/main.py [hunk 2]
```diff
@@ -20,6 +20,16 @@ def cmd_config(catalog_url):
         api.config(catalog_url)
 
 
+def cmd_verify(name, registry, top_hash, src, extra_files_ok):
+    pkg = api.Package.browse(name, registry, top_hash)
+    if pkg.verify(src, extra_files_ok):
+        print("Success")
```
Comment (akarve) [inline api/python/quilt3/main.py]:
```suggestion
        print("Verification succeeded")
```

**Comment by akarve** [comment]
@dimaryaz if it's not obvious then I'd say everything should be `--name`

File: api/python/quilt3/main.py [hunk 3]
```diff
@@ -77,9 +87,39 @@ def create_parser():
...
+        required=True,
+    )
+    verify_p.add_argument(
+        "--top-hash",
+        help="Hash of package to verify",
+        type=str,
+        required=True,
+    )
+    verify_p.add_argument(
+        "src",
```
Comment (akarve) [inline api/python/quilt3/main.py]: Maybe:
```suggestion
        "--dir",
```

**Pull request merged**